src/js/view/controls/controlbar.js

```diff
@@ -101,6 +101,17 @@ function buttonsInFirstNotInSecond(buttonsA, buttonsB) {
         !buttonsB.some(b => (b.id + b.btnClass === a.id + a.btnClass) && a.callback === b.callback));
 }
 
+function changeFullscreenTooltip(fullscreenButton, model, fullscreenTip) {
+    const localization = model.get('localization');
+    if (!model.get('fullscreen')) {
+        fullscreenTip.setText(localization.fullscreen);
```

Comment:
the logic in both conditional blocks are the same so let's move the commonality out of the conditions:
```
let fullscreenText = model.get('fullscreen') ? localization.exitFullscreen : localization.fullscreen;
fullscreenTip.setText(fullscreenText);
setAttribute(fullscreenButton.element(), 'aria-label', fullscreenText);
```